testing/mocker/example_mock_json/MRI-anatomy.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
A little nitpick for all the mock json files:
1. files to be renamed as `mock_` 
2. `_` instead of `-`

Reply:
Done